misc/scripts/install_d3d12_sdk_windows.py

```diff
@@ -6,16 +6,9 @@
 import sys
 import urllib.request
 
```

Comment:
This script was executable on purpose.

Reply:
Dagnabbit, Windows source control strikes again